.jshintrc

```diff
@@ -9,6 +9,8 @@
   "strict": "global",
   "undef": true,
   "unused": true,
+  "esversion": 6,
```

Comment:
We don't yet, no. I'll have a look to see what the options are for enabling this only for the Gulp scripts.